### DISCUSSION TITLE
Papers on science behind amrex

### DISCUSSION THREAD

**Discussion by cpraveen** [AMReX-Codes/amrex#2721] [General]
Is there a place which lists the papers which explain the methods behind amrex ?

**Comment by etpalmer63** [comment]
Are you looking for something like this? [AMReX: Block-structured adaptive mesh refinement for multiphysics applications](https://journals.sagepub.com/doi/full/10.1177/10943420211022811?casa_token=i5nuBUBrhzgAAAAA%3Awa9WIXAfBndm5vSe4sg9RrfQIPrJ3nIJnmhOGgyMZG-HlrAHgVSadoDHJNFe4epPC97i83r6n-W1uQ)

**Comment by cpraveen** [comment]
Thank you, that is useful to understand amrex. But to read about methods, e.g., transferring solution between grid levels, can you suggest some references.

**Comment by drummerdoc** [comment]
The issue Praveen is that the transfer of information across levels is really a function of the PDEs being solved, and thus isn't really an AMReX thing, per se.  AMReX provides a large set of tools to help, including FluxRegisters, interpolators and averaging operations.  So, you really need to look at the details of how various applications have implemented coarse-fine data transfers and operations.  For ideas, look at the advection and elliptic operators in the examples.

**Comment by cpraveen** [comment]
I will see the code. But if you can remember any papers on elliptic problems, and on hyperbolic problems, which work on the type of meshes used in amrex, please point them out. I am new to amr, so would like to first read something before delving into code.

**Comment by etpalmer63** [comment]
You may have already found it, but just in case: the [AMReX User's Guide](https://amrex-codes.github.io/amrex/docs_html/index.html) is a good place to start before going into the code.

**Comment by zingale** [comment]
additionally, a lot of application papers discuss the mathematical ideas they use for refluxing.  For Castro, the syncs for the gravity solve (Poisson equation) are described in this paper: https://ui.adsabs.harvard.edu/abs/2010ApJ...715.1221A/abstract

There are probably a dozen or so other AMReX-based code papers that will similarly describe things, so you might look for one that is related to your application area.

**Comment by cpraveen** [comment]
Thanks, these are helpful.
